docs/firstWorkload/README.md

```diff
@@ -418,7 +418,7 @@ can't guarantee that exactly these actions will be performed if
 "terraform apply" is subsequently run.
 ```
 
-Finally, since we are happy with these changes, we apply the Terraform template:
+Finally, since we are happy with these changes, we apply the Terraform template: Please confirm with "yes" for a prompt to perform the actions. 
```

Comment:
Consider removing colon after template since we have another sentence

Reply:
Done